app/src/main/java/com/cornellappdev/volume/ui/components/general/IndividualPublicationComponents.kt

```diff
@@ -0,0 +1,4 @@
+package com.cornellappdev.volume.ui.components.general
+
+class IndividualPublicationComponents {
```

Comment:
Is this in use?

Reply:
this is not in use now, but will be soon. I accidentally committed it oops :/